src/main/java/com/stripe/net/MultipartProcessor.java

```diff
@@ -59,20 +60,42 @@ public void addFormField(String name, String value) {
    * @param file File instance
    */
   public void addFileField(String name, File file) throws IOException {
-    String fileName = file.getName();
-    writer.append("--" + boundary).append(LINE_BREAK);
-    writer.append(
-        "Content-Disposition: form-data; name=\"" + name
-            + "\"; filename=\"" + fileName + "\"").append(
-        LINE_BREAK);
+    addFileField(name, file.getName(), new FileInputStream(file));
+  }
```

Comment:
Minor, but since these methods are meant to be internal anyway, can we just get rid of the `File` form of `addFileField` since it's only used in one place anyway? If other places in the library need to do this, they can instantiate their own `FileInputStream`.

Reply:
Done.